strawberry_django/mutations/resolvers.py

```diff
@@ -474,6 +474,7 @@ def update_m2m(
                 if data:
                     for k, inner_value in data.items():
                         setattr(obj, k, inner_value)
+                    obj.full_clean()
```

Comment:
I just noticed something. This needs to respect the `full_clean` options that are passed to the update/create methods

Reply:
Done in this commit: https://github.com/strawberry-graphql/strawberry-graphql-django/pull/405/commits/3f37550105b9bdc9ca157569871c346536e06a77 Thanks for mentioning.